next.orama.mjs

```diff
@@ -0,0 +1,23 @@
+import { Highlight } from '@orama/highlight';
+import { OramaClient } from '@oramacloud/client';
+
+import { DEFAULT_ORAMA_QUERY_PARAMS } from './next.constants.mjs';
+
+export const orama = new OramaClient({
+  endpoint: process.env.NEXT_PUBLIC_ORAMA_ENDPOINT,
+  api_key: process.env.NEXT_PUBLIC_ORAMA_API_KEY,
+});
+
+orama.startHeartBeat({ frequency: 3500 });
```

Comment:
is this value of any importance that is should be documented or a constant?

Reply:
Indeed, and documentation surrounding it would be great!

Reply:
Also, @micheleriva process.env shouldn't called directly from any other file besides next.constants.mjs, can you assign a constant for this process.env variables (and others) and also set default values? :)